manifests/init.pp

```diff
@@ -195,9 +195,9 @@
   $odoo_supported_versions = {
     'Debian' => {
       '9'  => ['11.0', '12.0'],
-      '10' => ['11.0', '12.0', '13.0', '14.0', '15.0', '16.0'],
-      '11' => ['14.0', '15.0', '16.0', 'system'],
```

Comment:
Please keep "system" in this list.  It is expected to work, and removing it from this list break the test that ensure it works:

> odoo class when using odoo version="system" works idempotently with no errors